lib/collection/src/operations/snapshot_ops.rs

```diff
@@ -21,24 +21,28 @@ pub enum SnapshotPriority {
     Snapshot,
     #[default]
     Replica,
+    // `ShardTransfer` is for internal use only, and should not be exposed/used in public API
+    #[serde(skip)]
```

Comment:
We could consider [`#[doc(hidden)]`](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#hidden) here as well. Though, we don't really expose `cargo doc` documentation for this.

Reply:
I don't think that would be useful. Pretty much the whole Rust codebase is "internal" and so the whole rust doc is "private" anyway. Why would we cut stuff from our own documentation? :D